service-api/src/main/java/greencity/service/NotificationService.java

```diff
@@ -0,0 +1,67 @@
+package greencity.service;
+
+import greencity.dto.notification.NotificationInternalDto;
+import greencity.dto.notification.NotificationDto;
+
+import java.util.List;
+
+public interface NotificationService {
+
+    public interface NotificationService {
+
```

Comment:
why is there a `NotificationService` nested inside a `NotificationService`?

```suggestion
public interface NotificationService {
```

Reply:
my fout